src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -115,7 +115,8 @@ private Node setMouseClickForNode(Node node, String milestoneName) {
     }
 
     private void handleMouseClick(String milestoneName) {
-        inputField.setText(milestoneName);
+        inputField.setDisable(true);
```

Comment:
Do we need to disable the inputField? Just wondering if suddenly user wants to change the milestone query.